pkg/authtoken/authtoken_test.go

```diff
@@ -0,0 +1,349 @@
+package authtoken
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"log"
+	"net/http"
+	"net/url"
+	"reflect"
+	"testing"
+)
+
+var (
+	a       byte = 116
```

Comment:
As mentioned in other comments we should avoid the use of global variables. Can we moved them where needed?